docs/cli.md

```diff
@@ -0,0 +1,158 @@
+This command supports many options, including the following; for the full list, run `mlrun build --help`:
+
+```text
+  --name TEXT            Function name
+  --project TEXT         Project name
+  --tag TEXT             Function tag
+  -i, --image TEXT       Target image path
+  -s, --source TEXT      Path/URL of the function source code - a PY file, or a directory to archive
+                         when using the -a|--archive option (default: './')
+  -b, --base-image TEXT  Base Docker image
+  -c, --command TEXT     Build commands; for example, '-c pip install pandas'
+  --secret-name TEXT     Name of a container-registry secret
+  -a, --archive TEXT     Path/URL of a target function-sources archive directory: as part of
+                         the build, the function sources (see -s|--source) are archived into a
+                         TAR file and then extracted into the archive directory 
+  --silent               Don't show build logs
+  --with-mlrun           Add the MLRun package ("mlrun")
+```
+
+> **Note:** For information about using the `-a|--archive` option to create a function-sources archive, see [Using a Sources Archive](#sources-archive) later in this tutorial.
+
+<a id="cli-cmd-run"></a>
+### The `run` Command
+
+Use the `run` CLI command to execute a task by using a local or remote function.
+This command supports many options, including the following; for the full list, run `mlrun run --help`:
+
+``` text
+  -p, --param key=val    Parameter name and value tuples; for example, `-p x=37 -p y='text'`
+  -i, --inputs key=path  Input artifact; for example, `-i infile.txt=s3://mybucket/infile.txt`
+  --in-path TEXT         Base directory path/URL for storing input artifacts
+  --out-path TEXT        Base directory path/URL for storing output artifacts
+  -s, --secrets TEXT     Secrets, either as `file=<filename>` or `env=<ENVAR>,...`; for example, `-s file=secrets.txt`
+  --name TEXT            Run name
+  --project TEXT         Project name or ID
+  -f, --func-url TEXT    Path/URL of a YAML function-configuration file, or db://<project>/<name>[:tag] for a DB function object
+  --task TEXT            Path/URL of a YAML task-configuration file
+  --handler TEXT         Invoke the function handler inside the code file
+```
+
+<a id="git-func"></a>
+## Building and Running a Function from a Git Repository
+
+To build and run a function from a Git repository, start out by adding a YAML function-configuration file in your local environment.
+This file should describe the function and define its specification.
+For example, create a **myfunc.yaml** file with the following content in your working directory:
+```yaml
+kind: job
+metadata:
+  name: remote-demo1
+  project: ''
+spec:
+  command: 'examples/training.py'
+  args: []
+  image: .mlrun/func-default-remote-demo-ps-latest
+  image_pull_policy: Always
+  build:
+    #commands: ['pip install pandas']
+    base_image: mlrun/mlrun:dev
```

Comment:
I don't know what this image is, please use `mlrun/mlrun:0.5.4` (I update the tags when I release)

Reply:
done